WordPressKit/WordPressKitTests/AccountServiceRemoteRESTTests.swift

```diff
@@ -404,7 +400,7 @@ class AccountServiceRemoteRESTTests: RemoteTestCase, RESTTestable {
         let expect = expectation(description: "Request WPCom Auth Link success")
 
         stubRemoteResponse(linkEndpoint, filename: requestLinkSuccessMockFilename, contentType: .ApplicationJSON)
-        remote.requestWPComAuthLink(forEmail: email, success: {
+        remote.requestWPComAuthLink(forEmail: email, clientID: "client123", clientSecret: "shhh", wpcomScheme: "wordpress", success: {
```

Comment:
Same as before, here and in the following uses, maybe we want to replace "wordpress" by a constant.